### PR TITLE
OpenAPI Credentials module

### PR DESCRIPTION
**Pull request by ThaddeusJiang**
- feat: 🎸 versions module
- chore: 🤖 version 0.0.8
- refactor: remove Enum from file_name
- OpenAPI Credentials module

### CODE REVIEW

File: openapi/openapi.yaml
Comment (Ko1103) [inline openapi/openapi.yaml:1]: version upしないとnpm packageがpublishされないです。

File: openapi/paths/credentials.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  operationId: getCredentials
+  security:
+    - tokenAuth: []
+  responses:
+    "200":
+      description: OK
+      content:
+        application/json:
+          schema:
+            $ref: ../components/schemas/CredentialsRole.yaml
```
Comment (Ko1103) [inline openapi/paths/credentials.yaml:19]: This should be Credentials.yaml.

File: openapi/components/schemas/PartyID.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+summary: party_id
+type: string
```
Comment (Ko1103) [inline openapi/components/schemas/PartyID.yaml:2]: maxLength 3

File: openapi/components/schemas/CountryCode.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+summary: country_code
+type: string
```
Comment (Ko1103) [inline openapi/components/schemas/CountryCode.yaml:2]: maxLength 2

File: openapi/paths/credentials.yaml [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    "401":
+      description: Unauthorized
+      $ref: ../components/responses/Unauthorized.yaml
```
Comment (Ko1103) [inline openapi/paths/credentials.yaml:46]: also 405 could be response.

File: openapi/paths/credentials.yaml [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+      application/json:
+        schema:
+          $ref: ../components/schemas/Credentials.yaml
+  responses:
+    "200":
+      description: OK
+      content:
+        application/json:
+          schema:
+            $ref: ../components/schemas/CredentialsRole.yaml
```
Comment (Ko1103) [inline openapi/paths/credentials.yaml:43]: > The credentials object in the response also contains extra information about the server such as its business details.

So I think credentialsObject should be returned, not credentialsRole.

File: openapi/paths/credentials.yaml [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
```
Comment (Ko1103) [inline openapi/paths/credentials.yaml:68]: Same as post, replace credentialsRole with credentialsObject.

File: openapi/paths/credentials.yaml [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
```
Comment (Ko1103) [inline openapi/paths/credentials.yaml:97]: objectが良さそう、ただ、他の会社の例だとresponse bodyがnoneのケースもある。
それでも良いと思う（その場合、200じゃなく204)

File: openapi/components/responses/Deleted.yaml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+description: Deleted
+content:
+  application/json:
+    schema:
+      type: object
+    example:
+      status_code: 202
```
Comment (Ko1103) [inline openapi/components/responses/Deleted.yaml:7]: 204?

File: openapi/components/responses/Deleted.yaml
Comment (Ko1103) [inline openapi/components/responses/Deleted.yaml:1]: This is common http status code known as No Content.
NoContent.yaml could be more useful and easy to understand.

Comment (ThaddeusJiang) [reply openapi/components/responses/Deleted.yaml:1]: @Ko1103 DELETE Method doesn't mean No Content, something just is `invalid` and `can no be used`.

# OCPI: 

CleanShot 2023-09-11 at 12 06 34@2x

# これは Coda API を参考して、202 にしました。
https://coda.io/developers/apis/v1#tag/Docs/operation/deleteDoc

CleanShot 2023-09-11 at 11 27 04

**Review by Ko1103 — APPROVED** [review]

**Pull request merged**